js/customize-snapshots.js

```diff
@@ -835,6 +837,149 @@
 		return date.valueOf();
 	};
 
+	component.conflict = {
+		warningTemplate: wp.template( 'snapshot-conflict' ),
+		icon: wp.template( 'snapshot-conflict-button' ),
+		refreshBuffer: 250,
+		controls: {},
+		pendingRequest: {},
+		notificationCode: 'snapshot_conflict',
+		notificationTemplate: wp.template( 'snapshot-notification-template' )
+	};
+
+	/**
+	 * Handles snapshot conflict events.
+	 *
+	 * @return {void}
+	 */
+	component.handleEventForConflicts = function() {
+		api.state( 'saved' ).bind( function( saved ) {
+			if ( saved ) {
+				_.each( component.conflict.controls, function( control ) {
+					control.remove();
+				} );
+				if ( component.conflict._currentRequest ) {
+					component.conflict._currentRequest.abort();
+					component.conflict._currentRequest = null;
+				}
+				component.conflict.controls = {};
+				component.conflict.pendingRequest = {};
+
+				wp.customize.control.each( function( control ) {
```

Comment:
Use `api` instead of `wp.customize` in this case and the other cases in this PR.